build/release/README.md

```diff
@@ -19,8 +19,7 @@ the release binaries following these steps:
 
 1. `git clone https://github.com/lightningnetwork/lnd.git`
 2. `cd lnd`
-3. `./build/release/release.sh <TAG> # <TAG> is the name of the next
```

Comment:
what if we moved this to `docs`? it's probably harder to find buried in a subpackage